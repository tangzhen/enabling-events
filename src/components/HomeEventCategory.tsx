import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
import Tab from "@material-ui/core/Tab";
import Tabs from "@material-ui/core/Tabs";
import Typography from "@material-ui/core/Typography";
import React from "react";

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      marginBottom: theme.spacing(5),
    },
    title: {
      fontWeight: "bold",
      marginTop: theme.spacing(6),
      marginLeft: theme.spacing(9),
    },
  })
);

export default function HomeEventCategory() {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Typography variant="h5" color="primary" className={classes.title}>
        Popular Events
      </Typography>
      <Tabs
        value={2}
        indicatorColor="primary"
        textColor="primary"
        aria-label="disabled tabs example"
        variant="fullWidth"
      >
        <Tab label="ALL" />
        <Tab label="FOR YOU" />
        <Tab label="TODAY" />
        <Tab label="THIS WEEKEND" />
        <Tab label="FREE" />
        <Tab label="FOOD & DRINK" />
        <Tab label="ENTERTAINMENT" />
      </Tabs>
    </div>
  );
}
