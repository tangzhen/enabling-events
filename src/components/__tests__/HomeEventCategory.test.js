import React from "react";
import { render } from "../../../test/test-utils";
import HomeEventCategory from "../HomeEventCategory";

describe("HomeEventCategory", () => {
  it("should render component correct", () => {
    const { container } = render(<HomeEventCategory />);
    expect(container).toMatchSnapshot();
  });
});
