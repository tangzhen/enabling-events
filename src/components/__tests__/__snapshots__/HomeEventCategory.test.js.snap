// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeEventCategory should render component correct 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <h5
      class="MuiTypography-root makeStyles-title-2 MuiTypography-h5 MuiTypography-colorPrimary"
    >
      Popular Events
    </h5>
    <div
      class="MuiTabs-root"
    >
      <div
        class="MuiTabs-scroller MuiTabs-fixed"
        style="overflow: hidden;"
      >
        <div
          aria-label="disabled tabs example"
          class="MuiTabs-flexContainer"
          role="tablist"
        >
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              ALL
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              FOR YOU
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="true"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected MuiTab-fullWidth"
            role="tab"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              TODAY
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              THIS WEEKEND
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              FREE
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              FOOD & DRINK
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth"
            role="tab"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              ENTERTAINMENT
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <span
          class="PrivateTabIndicator-root-3 PrivateTabIndicator-colorPrimary-4 MuiTabs-indicator"
          style="left: 0px; width: 0px;"
        />
      </div>
    </div>
  </div>
</div>
`;
